posthog/api/feature_flag.py

```diff
@@ -175,7 +179,7 @@ def _update_filters(self, validated_data):
             validated_data["filters"] = validated_data.pop("get_filters")
 
 
-class FeatureFlagViewSet(StructuredViewSetMixin, viewsets.ModelViewSet):
+class FeatureFlagViewSet(StructuredViewSetMixin, ForbidDestroyModel, viewsets.ModelViewSet):
```

Comment:
Maybe this is covered by a mixin but does this not mean that the flag will be returned in GET requests but just with a property of `deleted: true` or is there something elsewhere already filtering those out?

Reply:
But, they're not handled uniformly... I guess they should be but that could be a follow-up